http-clients/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     compile libs.wsRsApi
 
     testCompile project(':http-servers')
+    testCompile files('lib/VersionTest.jar')
```

Comment:
You could make this an unpublished project with a fixed version…but it'd be yet one more project.